DTXManiaプロジェクト/コード/全体/CConfigIni.cs

```diff
@@ -2200,13 +2245,30 @@ public void tファイルから読み込み( string iniファイル名 )
 											{
 												this.b演奏情報を表示する = C変換.bONorOFF( str4[ 0 ] );
 											}
-                                            else if( str3.Equals( "ChipVolume" ) )
+											else if( str3.Equals( "AutoChipVolume" ) )
```

Comment:
ChipVolume is gone entirely (it didn't work.) AutoChipVolume is deprecated (it barely worked.) The new code, though, will upgrade loaded config.ini AutoChipVolume settings into SongPlaybackLevel config settings, since that what AutoChipVolume controlled in the rare cases where it even worked (due to the previously mentioned interactions with the "sound monitor" code.)